analysis/src/python/data_analysis/analysis/timing/time_compare.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+import logging
+import os
+import sys
+import time
+from pathlib import Path
+from typing import List
+
+import numpy as np
+import pandas as pd
+from hyperstyle.src.python.review.application_config import LanguageVersion
+from hyperstyle.src.python.review.reviewers.common import LANGUAGE_TO_INSPECTORS
+
+from analysis.src.python.data_analysis.model.column_name import SubmissionColumns
+from analysis.src.python.data_analysis.utils.df_utils import read_df
+from analysis.src.python.data_analysis.utils.logging_utils import configure_logger
+from analysis.src.python.evaluation.common.file_util import create_file
+from analysis.src.python.evaluation.common.parallel_util import run_and_wait
+
+TEMPLATE_FOLDER = Path(__file__).parents[4] / 'resources' / 'evaluation' / 'qodana' / 'project_templates'
+PROFILE_FOLDER = Path(__file__).parents[4] / 'resources' / 'evaluation' / 'qodana' / 'inspection_profiles'
```

Comment:
It looks strange? Can we add and use a const SRC_FOLDER or ROOT_FOLDER on the root?